app/src/main/java/org/listenbrainz/android/presentation/features/yim/YimViewModel.kt

```diff
@@ -44,9 +44,11 @@ class YimViewModel @Inject constructor(private val repository: YimRepository, @A
     
     private fun getData() {
         viewModelScope.launch {
-            val response = async {  username?.let { repository.getYimData(username = it)}!! }
-            when (response.await().status){
-                Resource.Status.SUCCESS -> yimData.value = response.await()
+            val response = username?.let {
+                    repository.getYimData(username = it)
+                }!!
```

Comment:
I am not eyeing too much on the force unwraps here. But could these be avoided in general?

Reply:
My bad looks like I'm watching to many youtube videos xD.
Ill inline these type of cases.

Reply:
Done